frontend/app/components/work-packages/wp-breadcrumb/wp-breadcrumb.directive.html

```diff
@@ -0,0 +1,11 @@
+<div class="wp-breadcrumb -show"
+     ng-if="workPackage.ancestors.length > 0">
+  <ul class="breadcrumb">
+    <li ng-repeat="ancestor in workPackage.ancestors" class="icon-context icon-small icon-arrow-right5">
+      <a title="{{ancestor.name}}"
```

Comment:
Prefer `ng-attr-title="{{ ancestor.name }}"` over `title={{..}}` so that the attribute is only set once the value is ready.